cmd/root.go

```diff
@@ -17,7 +17,8 @@ var issuerURL string
 var roleARN string
 
 const (
-	flagVerbose = "verbose"
+	flagVerbose    = "verbose"
+	successMessage = "Signed in successfully! You can now return to CLI."
```

Comment:
@crohacz had a nice message over https://github.com/chanzuckerberg/go-misc/pull/64/files#diff-9d58a4e5129173a25eea5c5df7bbaac9L118. Maybe we can re-use it here?